ui/perfherder/compare/CompareTable.jsx

```diff
@@ -164,7 +164,7 @@ export default class CompareTable extends React.PureComponent {
                             this.getHashBasedId(testName, rowLevelResults.name),
                           )
                         }
-                        title="Permalink to this test"
+                        title={`Permalink to this test: ${rowLevelResults.name}`}
```

Comment:
Suggested wording: `Permalink to test ${rowLevelResults.name}` and for the same text above.